### DISCUSSION TITLE
Trying to Uninstall by script in GPO[Q&A]

### DISCUSSION THREAD

**Discussion by kidylax** [glpi-project/glpi-agent#617] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hello everyone, 
as a technician looking to enhance my skills, I would like to clean up our GLPI environment. Therefore, I have created a GPO to deploy the GLPI-Agent. Unfortunately, I cannot find a way to uninstall the agent if it is not at the correct version. I believe I may have overlooked a line in the documentation, but I have tried several methods, and none have worked. I assume that those with existing infrastructures have figured out how to do this. Thank you in advance.

**Comment by g-bougard** [comment, marked as answer]
Hi @kidylax 

you should use the [vbs script](https://raw.github.com/glpi-project/glpi-agent/develop/contrib/windows/glpi-agent-deployment.vbs) in your GPO as explained in documentation here: https://glpi-agent.readthedocs.io/en/latest/installation/index.html#large-installations

This script has few advantages like preventing some installation issues and the agent will be replaced by the version you want.

**Reply by kidylax** [reply]
Thank you indeed i missed this, i didn't understand the replacement.
Great day !
